dark-mode.css

```diff
@@ -168,7 +168,7 @@ body.dark-mode {
 /***************Main Chat*****************/
 
 body.dark-mode a {
-	color: var(--color-light-blue);
+	color: var(--info-font-color);
```

Comment:
Why change link color from `--color-light-blue` to `--color-gray-light`?

Reply:
This is mistake, I have fixed it to `--color-light-blue`